src/Symfony/Bundle/SecurityBundle/Tests/DependencyInjection/Compiler/RegisterGlobalSecurityEventListenersPassTest.php

```diff
@@ -45,6 +49,42 @@ protected function setUp(): void
         $securityBundle->build($this->container);
     }
 
+    /**
+     * @dataProvider providePropagatedEvents
+     */
+    public function testEventIsPropagated(string $configuredEvent, string $registeredEvent): void
```

Comment:
I guess it shouldn't be `void`

Reply:
Correct, please remove `: void` here.

Reply:
Why shouldn't it be void? It's a test method, it doesn't return anything, therefore it has a void return type. Am I missing something?

Reply:
You are right, but this is our current code style. We can probably debate to change it, but for this PR, we should maintain the current code style for consistency.

Reply:
Returning void is correct, however we do not add void return types to the test classes (to avoid useless clutter).

Reply:
AFAIK this should be reported by fabbot.io now

Reply:
Ok, well then, I just removed the `void` return type.